### DISCUSSION TITLE
let contributors join "teams" or "pools"

### DISCUSSION THREAD

**Discussion by andrewm4894** [LAION-AI/Open-Assistant#3432] [Ideas]
could be interesting to let contributors form their own "groups" or "pools" or could be tag based. such that contributors can compare and rank within their different groups or pools or tags.

eg say i want to make a little group within my job and do the gamification stuff within this group as opposed to just globally. 

or maybe something like a "sprint" or "competition" that could be released centrally if/when maybe we want to try juice volume of some specific subsets of tasks and datasets. 

maybe is some more general way to implement something that would give both - maybe a simple tag system could work

**Comment by fozziethebeat** [comment]
This is a good idea but not something that's well beyond the MVP.  I've marked this as needs discussion since we have to figure out exactly what users would want as groups.  Before doing this I think we should start collecting data from real people and then poll them to see what's going to be most beneficial to their contribution.
